### DISCUSSION TITLE
Crash on start

### DISCUSSION THREAD

**Discussion by NullLurker** [ErinaSugino/Starbound-SBR-Mod#131] [Trouble Shooting] [answered]
My Starbound crashes before even making it to the main menu, I think the crash is related to the Aphrodite's Bow mod but I can't be sure

[starbound.log](https://github.com/user-attachments/files/16897632/starbound.log)

**Comment by ErinaSugino** [comment, marked as answer]
You wrongly installed Sexbound and the SBR Aphrodite's Bow patch - not at all for as far as the game is concerned - so Aphrodite's Bow itself, which is the only correctly installed mod, crashes your game because it depends on Sexbound, which is not installed.

**Reply by NullLurker** [reply]
Thank you very much for your answer

**Reply by NullLurker** [reply]
What would be the correct way should i leave them in the .zip they came in or fully take everything out of the Folders?

**Reply by ErinaSugino** [reply]
https://github.com/ErinaSugino/Starbound-SBR-Mod/wiki/Using-the-mod#installation
